src/utils/FilterHelpers.ts

```diff
@@ -1,7 +1,11 @@
-import { IFilter, FilterDimension } from "../interfaces/IFilter";
+import { IFilter } from "../interfaces/IFilter";
 
 export const determineClassName = (filter: IFilter) => {
-  switch (filter.dimension) {
+  // TODO_GENERALIZATION This is this tailwind class name thingy, to be sorted out.
+  // For now, we're lucky: The classes are just 'propertyName'+'-filter' ...
+  return filter.propertyName+"-filter";
```

Comment:
NIT: You can use a template string here rather than concatenation: 

```
return `${filter.propertyName}-filter`;
```

These are generally a bit easier to read but again, that's subjective which is why this is a nitpick.